src/components/shared/Entity/LogDialogActions.tsx

```diff
@@ -13,35 +12,51 @@ interface Props {
     };
 }
 
+const buttonSX: SxProps<Theme> = {
```

Comment:
**Merge Conflict Source:** Perfectly fine if you override my changes to `LogDialog` and `LogDialogActions`. As far as `buttonSX` is concerned, two comments come to mind:

1. The `borderRadius` of the global `Button` style will need to be _reapplied_ since these buttons are using a custom `sx` constant.

1. To stick with the existing naming convention, the constant should be named `buttonSx`. Not a big deal and you can choose whether or not to rename it.

Reply:
Removed the `sx` based styling and adding the space around the buttons with a stack component